lib/array-store.js

```diff
@@ -9,15 +9,14 @@ var ArrayStore = function(datasetId, data) {
   this.topic = {};
   this.subscription = {};
   this.datasetId = datasetId;
-}
+};
 
-ArrayStore.prototype.create = function(object, ts) {
+ArrayStore.prototype.create = function(object) {
   var self = this;
   var deferred = q.defer();
   setTimeout(function() {
     object.id = shortid.generate();
     self.data.push(object);
-    console.log('Created object:', object);
```

Comment:
For the moment I think they can stay. There is a JIRA to introduce logger support to the modules so after that I think we can switch the `console` warnings to errors.